taskfiles/ci.yml

```diff
@@ -3,6 +3,9 @@ version: '3'
 vars:
   DOCKER_REGISTRY: '778696301129.dkr.ecr.us-west-2.amazonaws.com'
 
+env:
+  KUBECONFIG: /work/kubeconfig
```

Comment:
Why are we setting `KUBECONFIG`?

Reply:
Because when I created `kind` cluster something goes side ways and didn't populate kubeconfig to the `go test`, so that test case failed as it couldn't reach K8S API server. I will try to remove in the next few git pushes.